### PR TITLE
Add FOREMAN_DB_TYPE variable to change --foreman-db-type

### PR DESCRIPTION
**Pull request by domcleal**

### CODE REVIEW

File: fb-install-foreman.bats [hunk 1]
```diff
@@ -137,7 +137,10 @@ EOF
 }
 
 @test "run the installer" {
-  args="--no-colors -v ${FOREMAN_USE_LOCATIONS:+--foreman-locations-enabled=$FOREMAN_USE_LOCATIONS} ${FOREMAN_USE_ORGANIZATIONS:+--foreman-organizations-enabled=$FOREMAN_USE_ORGANIZATIONS}"
+  args="--no-colors -v"
+  [ -n "${FOREMAN_USE_LOCATIONS}" ] && args="${args} --foreman-locations-enabled=${FOREMAN_USE_LOCATIONS}"
```
Comment (ekohl) [inline fb-install-foreman.bats]: nitpick: doesn't bash support ```+=```?

Comment (domcleal) [inline fb-install-foreman.bats]: So it does, thanks!

File: fb-install-foreman.bats [hunk 2]
```diff
@@ -137,9 +137,12 @@ EOF
...
 
 @test "run the installer" {
-  args="--no-colors -v ${FOREMAN_USE_LOCATIONS:+--foreman-locations-enabled=$FOREMAN_USE_LOCATIONS} ${FOREMAN_USE_ORGANIZATIONS:+--foreman-organizations-enabled=$FOREMAN_USE_ORGANIZATIONS}"
-  [ x$FOREMAN_VERSION = "x1.7" -o x$FOREMAN_VERSION = "x1.8" ] || args="${args} --foreman-logging-level=debug"
-  foreman-installer --foreman-admin-password=admin $args
+  args="--no-colors -v --foreman-admin-password=admin"
+  [ -n "${FOREMAN_USE_LOCATIONS}" ] && args+=" --foreman-locations-enabled=${FOREMAN_USE_LOCATIONS}"
+  [ -n "${FOREMAN_USE_ORGANIZATIONS}" ] && args+=" --foreman-organizations-enabled=${FOREMAN_USE_ORGANIZATIONS}"
+  [ -n "${FOREMAN_DB_TYPE}" ] && args+=" --foreman-db-type=${FOREMAN_DB_TYPE}"
+  [ x$FOREMAN_VERSION = "x1.7" -o x$FOREMAN_VERSION = "x1.8" ] || args+=" --foreman-logging-level=debug"
```
Comment (ekohl) [inline fb-install-foreman.bats]: I think the conditional can be dropped here, but I'm fine if that's a separate change.

**Comment by ekohl** [comment]
:+1:

File: fb-install-foreman.bats (hunk 2 above)
Comment (domcleal) [inline fb-install-foreman.bats]: Yeah, I'll go through and clear out the old FOREMAN_VERSION support some time.. we only need to support as far back as the currently updated stable really.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**